chart/values.yaml

```diff
@@ -615,6 +615,22 @@ nats:
   natsbox:
     enabled: false
 
+stats:
+  # -- Enable stats
+  enabled: true
```

Comment:
Should this also be enabled by eventing.enabled?
Wdyt @tiagolobocastro @chriswldenyer @avishnu ?